src/game/server/swarm_sdk_server.vcxproj

```
@@ -234,9 +236,9 @@ if ERRORLEVEL 1 exit 1
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <AdditionalLibraryDirectories>..\..\lib\common\.;..\..\lib\public\.;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
       <IgnoreSpecificDefaultLibraries>libc;libcd;libcmtd;%(IgnoreSpecificDefaultLibraries)</IgnoreSpecificDefaultLibraries>
-      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <GenerateDebugInformation>false</GenerateDebugInformation>
```

```suggestion
      <GenerateDebugInformation>true</GenerateDebugInformation>
```

we keep debug information around for release builds for figuring out minidumps

---

done.